docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Docker image for c2rust CI and containerized development.
 # Multi-distro via optional BASE_IMAGE argument.
-# RUST_VER argument should be set to the contents of the rust-toolchain file in
+# RUST_VER argument should be set to the contents of the rust-toolchain.toml file in
```

Comment:
This seems wrong--`RUST_VER` should probably still be a version string, not a TOML document.

Reply:
Oh yeah, it should be "set to the contents of the channel in the rust-toolchain.toml file", right?

Reply:
Fixed in https://github.com/immunant/c2rust/pull/520/commits/48a9927ab2fdfd785d49dd60e6f956b387dfdef0.